admin/load.php

```diff
@@ -126,18 +126,25 @@ function perflab_render_modules_page() {
  * @param array  $module_settings Associative array of the module's current settings.
  */
 function perflab_render_modules_page_field( $module_slug, $module_data, $module_settings ) {
-	$base_id   = sprintf( 'module_%s', $module_slug );
-	$base_name = sprintf( '%1$s[%2$s]', PERFLAB_MODULES_SETTING, $module_slug );
-	$enabled   = isset( $module_settings['enabled'] ) && $module_settings['enabled'];
+	$base_id         = sprintf( 'module_%s', $module_slug );
+	$base_name       = sprintf( '%1$s[%2$s]', PERFLAB_MODULES_SETTING, $module_slug );
+	$enabled         = isset( $module_settings['enabled'] ) && $module_settings['enabled'];
+	$can_load_module = perflab_can_load_module( $module_slug );
```

Comment:
@felixarntz When I see `can load module` it always feels like there are might be some limitations why the module can't be loaded which is not the case in our situation because what we check here is whether the module is implemented in the core or not. Maybe we should rename this function to be `perflab_module_is_implemented_in_core` or something similar to make it clear what it does?

Reply:
@eugene-manuilov That would limit the use-case more than needed. This closure could be used for other checks than just whether this exists in core. We could even have something like a module that only works in PHP 7 just to make an example. So we shouldn't limit ourselves artificially.

I'm certainly open to finding a better name, but it shouldn't be tied to whether the feature is available in core.

Reply:
Ah.. ok, thanks for explaining, @felixarntz.